tools/c7n_azure/c7n_azure/templates/dedicated_functionapp.parameters.json

```diff
@@ -18,13 +18,13 @@
             "value": "cloud-custodian"
         },
         "sku": {
-            "value": "BASIC"
+            "value": "STANDARD"

```

Comment:
Rather than feeding these params through a parameters.json file can we pass them in when you make the deployment call and just override the defaults? If so, we can move these values to constants too.